doc-independent-rosindex.yaml

```diff
@@ -0,0 +1,20 @@
+%YAML 1.1
+# ROS buildfarm doc-build file
+---
+documentation_type: external_site
+doc_repositories:
+- https://github.com/ros2/rosindex.git
+jenkins_job_priority: 90
```

Comment:
we'll want to double check these priority and timeout parameters before merging.

Reply:
Priority for sure, timeout was set after @tfoote [reported 90 minutes to build the whole site](https://github.com/ros2/rosindex/pull/54#issuecomment-424290434).I took a 30 minutes buffer, just in case.